File: MEAN/JSFun/fizzbuzz.js
function fizzbuzz(n){
    var newArr=[]
    if(typeof(n) !== 'number'){
        console.log("Parameter must be a positive number");
        return null;
    }
    else{
        for(var i=1; i<n+1;i++){
            if(i%15==0){
                console.log("FizzBuzz");
                newArr.push("FizzBuzz")
            }
            else if(i%3==0){
                console.log("Fizz")
                newArr.push("Fizz")
            }
            else if(i%5==0){
                console.log("Buzz")
                newArr.push("Buzz")
            }
            else {
                console.log(i)
                newArr.push(i);
            }
        }
        return newArr;
    }
}

console.log(fizzbuzz(15))
console.log(fizzbuzz('fifteen'))

File: JAVA/JavaFun/FizzBuzz.java
/**
 * FizzBuzz
 */
public class FizzBuzz {

    public String fizzBuzz(int number) {
        // String ans;

        if(number%15==0){
            return "FizzBuzz";
            // ans.concat("FizzBuzz");
        }
        else if(number%3==0){
            return "Fizz";
        }
        else if(number%5==0){
            return "Buzz";
        }
        else {
            return Integer.toString(number);
        }
        // return ans;
    }
}